notNeededPackages.json

```diff
@@ -5839,6 +5839,10 @@
             "libraryName": "stripe",
             "asOfVersion": "8.0.0"
         },
+        "stripe-v3": {
+            "libraryName": "@stripe/stripe-js",
+            "asOfVersion": "2.4.0"
```

Comment:
I don't think this is allowed at the moment; you may just need to delete the types and not edit this file.